serverless/messaging/concepts.mdx

```diff
@@ -165,6 +185,15 @@ The message queuing model provides a pattern or framework for sending messages,
 
 </Concept>
 
+
+<Concept>
+
+## Queue volume
+
+Queue volume is one of the factors affecting the billing of Scaleway Messaging and Queuing with SQS. Queue volume is calculated as the number of messages in a queue * the message size. Or, the sum of the size of all messages in a queue.
```

Comment:
I had a hard time understanding what the "*" meant, you could maybe use the word "times" or "multiplied by"?